operator/controllers/kyma_controller.go

```diff
@@ -416,6 +421,8 @@ func (r *KymaReconciler) SetupWithManager(mgr ctrl.Manager, options controller.O
 		}
 	}
 
+	controllerBuilder = listener.RegisterListenerComponent(logger, mgr, controllerBuilder, listenerAddr, &handler.EnqueueRequestForObject{}, componentName)
```

Comment:
why do you need explicit enqueueforobject here, wouldn't that be done through your handler?

Reply:
The Event Handler after the last refactor is a parameter which the caller of the `RegisterListenerComponent()` method should specify for future proofing. For the time being, it is a simple `EnqueueRequestForObject{}` for the `kyma` and `manifest` operators.

Reply:
Hmm, I really think this is bad practice to handle the manager around and then to the enqueue inside. I think you should instead work with the controllerBuilder (if at all)